server/config/topic_page/poverty/USA.textproto

```diff
@@ -12,8 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-## This file specifies the config for topic page
+## This file specifies the topic page config for poverty in USA
 
+metadata {
+  topic_id: 'poverty'
+  topic_name: 'Poverty'
+  place_dcid: 'country/USA'
+  place_dcid: 'geoId/06'
```

Comment:
what is geoId/06 used for here?

Reply:
we should specify all pages that the topic page works for. i added CA for testing, and also hoping that the config works for both country and state..